src/main/java/com/fcgl/madrid/points/model/AuditModel.java

```diff
@@ -0,0 +1,45 @@
+package com.fcgl.madrid.points.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+@JsonIgnoreProperties(
+        value = {"createdAt", "updatedAt"},
+        allowGetters = true
+)
+public abstract class AuditModel implements Serializable {
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "created_at", nullable = false, updatable = false)
+    @CreatedDate
+    private Date createdAt;
```

Comment:
This is what the `createdAt` and `updatedAt` look like now
`"createdAt": "2019-09-06T12:39:37.370Z"`
`"updatedAt": "2019-09-06T12:39:37.370Z"`

Why the deviation from epoch time? And is this timestamp based on server time or some other time?

Reply:
This holds exactly the same data just in a time format and handled by JPA so it will be the most accurate and close to correctness.

Reply:
Sounds good, will probably implement something similar for the other services. Thanks :)